boards/samr21-xpro/include/periph_conf.h

```diff
@@ -161,12 +165,16 @@ static const uart_conf_t uart_config[] = {
         .rx_pin   = GPIO_PIN(PA,23),
         .tx_pin   = GPIO_PIN(PA,22),
 #ifdef MODULE_PERIPH_UART_HW_FC
-        .rts_pin  = GPIO_UNDEF,
-        .cts_pin  = GPIO_UNDEF,
+        .rts_pin  = GPIO_PIN(PA,22),
```

Comment:
This is the same pin as the tx pin. Does it work like this ?

Reply:
ups pushed wrongly

Reply:
fixed, sorry for that